packages/ui/react-ui/src/CrossmintPayButton.tsx

```diff
@@ -42,6 +42,9 @@ export const CrossmintPayButton: FC<CrossmintPayButtonReactProps> = ({
     prepay,
     locale = "en-US",
     currency = "USD",
+    layout = "popUpWindow",
+    successCallbackURL = "https://www.crossmint.com/console/",
```

Comment:
After the payment succeeded or failed, the user would be redirected here directly, or would see the congrats with some action button?

Reply:
I think we could redirect for the initial version, custom buttons could be defined later